### ISSUE TITLE
Allow custom script steps in packaging phase?

### ISSUE THREAD

**Issue by djMax** [CocoaPods/cocoapods-packager#127]
I have a project which uses some generated files as part of the build. Right now, I have to make a whole separate git repo with these generated files in them in order to package. It would be nice if I could have a script that runs during the packaging phase, but I don't see a way to do this now.

**Comment by neonichu** [comment]
I believe any defined `prepare_command` (see <https://guides.cocoapods.org/syntax/podspec.html#prepare_command>) will be executed when packaging. Does this help?

**Comment by djMax** [comment]
I don't think that runs during packaging - at least it didn't for me. Maybe I did something else wrong, so if "it really should" then I'll dig more.

**Comment by neonichu** [comment]
Since a `pod install` is basically part of the process, they *should* run, but I haven't tested it myself at all.
